### ISSUE TITLE
PNG export of breadboard view shows pixel artifacts across bottom of image

### ISSUE THREAD

**Issue by chuank** [fritzing/fritzing-app#3261]
Purely cosmetic, but still something that impacts the exporting of clean Fritzing images.

When exporting breadboard views in PNG format, there is sometimes one or more horizontal rows of additional pixels at the bottom of the image that's not part of the original view. It looks like leftover pixels from an image buffer. This does not seem to impact the JPG exporter.

Here are some examples:
![image](https://cloud.githubusercontent.com/assets/1781911/16903666/bd66c0ba-4cc7-11e6-8081-9520bf407466.png)

![image](https://cloud.githubusercontent.com/assets/1781911/16903675/d38fa5aa-4cc7-11e6-90b2-c6a723f7a265.png)

I'm using the OSX version of the application.

**Comment by EdJoPaTo** [comment]
got the same on the right side of the exported image (but not on the bottom)

on Arch Linux installed via AUR, v0.9.3

**Comment by thomasw-mitutoyo-ctl** [comment]
> got the same on the right side of the exported image (but not on the bottom)
I have a single line of transparent pixels at the bottom.
0.9.4 on Windows 10

**Comment by KjellMorgenstern** [comment]
Breadboard fz
[wirec3.zip](https://github.com/fritzing/fritzing-app/files/5397966/wirec3.zip)

Exprot before #3725
![wirec3_bb](https://user-images.githubusercontent.com/6494025/96376595-35bd0c00-1180-11eb-8760-2c4d798a933f.png)

Export after #3725
![wirec3_bb2](https://user-images.githubusercontent.com/6494025/96376597-3786cf80-1180-11eb-90d0-65a0cde18831.png)

So, artifacts at the right are gone.
However there is still a transparent line at the bottom that could look like reported by @chuank , depending on where the image is used.

It is a bit difficult to show those transparent lines, I created a codepen: https://codepen.io/team/fritzing/pen/MWeejyP 
 Note the gap between the images.

**Comment by failiz** [comment]
Strange, I open those images in Gimp (size is 366x282) and I could not find the transparency. The alpha channel was set to 255 in all the pixels. Could it be an artifact of the viewer?

**Comment by thomasw-mitutoyo-ctl** [comment]
Other machine here, does not produce a transparent bottom line. Version 0.9.4  (CD-498-0-a1ffcea 2019-12-01) 64 [Qt 5.12.3] Windows 10. Could this be a library/DLL issue?

**Comment by KjellMorgenstern** [comment]
My mistake, the transparent line is not there.
